pkgs/applications/video/corrscope/default.nix

```diff
@@ -1,31 +1,28 @@
 { lib
 , mkDerivationWith
 , python3Packages
+, fetchurl
 , wrapQtAppsHook
 , ffmpeg
 , qtbase
 }:
 
 mkDerivationWith python3Packages.buildPythonApplication rec {
   pname = "corrscope";
-  version = "0.7.0";
+  version = "0.7.1";
 
-  src = python3Packages.fetchPypi {
-    inherit pname version;
-    sha256 = "0m62p3jlbx5dlp3j8wn1ka1sqpffsxbpsgv2h5cvj1n1lsgbss2s";
+  src = fetchurl {
+    url = "https://github.com/corrscope/corrscope/releases/download/0.7.1/corrscope-${version}-py3-none-any.whl";
+    sha256 = "0xc6mp4b5msp4rgbzlvjypm3w1vw41r9h5hxvcw25lgj5h2yimhv";
   };
 
-  postPatch = ''
-    substituteInPlace setup.py \
-      --replace 'attrs>=18.2.0,<19.0.0' 'attrs>=18.2.0' \
-      --replace 'numpy>=1.15,<2.0,!=1.19.4' 'numpy>=1.15,<2.0'
-  '';
+  format = "wheel";
```

Comment:
https://github.com/corrscope/corrscope/commit/a3e779e13cf3d954737cc221755dd4222e6719d6#commitcomment-54263064

Reply:
source would be highly perferred, not sure how difficult it would be to just build from source

Reply:
It's built with poetry, I couldn't figure out how to package that so I initially went with pypi instead. I can check if something about out poetry support improved.

Reply:
I think we should build this from source, too. Shouldn't be that hard and probably just requires setting the format to poetry.

Reply:
`poetry2nix` was as confusing & not-working as before, but using `poetry-core` & pulling in an upstream patch works.